### DISCUSSION TITLE
I'm Newbie for Github , now im completing implement GhostManager but i cant login with default user-password from Docs

### DISCUSSION THREAD

**Discussion by lDespair** [GhostManager/Ghostwriter#248] [Q&A]
I'm Newbie for Github , now im completing implement GhostManager but i cant login with default user-password from Docs

**Comment by chrismaddalena** [comment]
Hi @lDespair,

The `ghostwriter-cli` utility generates a random password for the initial administrator user. It's printed at the end of the `./ghostwriter-cli install` command. If you need it again, you can run `./ghostwriter-cli config get admin_password` to retrieve it. Once you login for the first time, you can change the password or make a new account to use.

It's all detailed here: https://www.ghostwriter.wiki/getting-started/quickstart
